extra_data/export.py

```diff
@@ -258,6 +287,12 @@ def main(argv=None):
         "--key", help="Stream only matching keys ('*' is a wildcard)",
         default='*',
     )
+    ap.add_argument(
+        "--data-like", help="Optional module stacking: online -> stack module \
+           sources into a single source (default: file -> keep modules \
+           separate as in files)",
+        default='file', choices=['file', 'online']
```

Comment:
I'd call this something like `--append-detector-modules`. It's possible to connect a bridge to an individual module instead of the appender, and we've considered the possibility of using the TrainMatcher (which would send a message with 16 arrays for the separate modules) instead of the combiner. So it's best not to suggest that one specific format is the 'online' format. But of course we can document that it's similar to a common setup for online data.

Reply:
No problem. In the first draft I took over from the karabo-bridge-server-sim implementation, we can discuss tomorrow how to adapt and generalize this in the best way.

Reply:
The use case for the server sim is a bit different. The data sent there is always a stack of all modules, only the shape changes.

Reply:
I changed the name to ``--detector-modules`` and the choices to ``file`` (default) and ``stack``. While this is not a generalization yet, it stresses more clearly what this particular option is for.

Reply:
Nitpicking, but @takluyver suggestion is clearer (`append-detector-module`) as it indicate that all modules are aggregated in a single array. I'd also suggest to not have choices, they don't really make sens in that context. Just the option without argument should be enough (present: modules are aggregated, not present: each module is a separate datasource)

Reply:
O.k. I see your point